Python-3.14.2/Apple/testbed/Python.xcframework/build/utils.sh

```
@@ -0,0 +1,151 @@
+# Utility methods for use in an Xcode project.
```

Shellcheck (0.11.0)

[error] Tips depend on target shell and yours is unknown. Add a shebang or a 'shell' directive.

(SC2148)